app.py

```diff
@@ -108,6 +108,16 @@ async def submit(request: Request) -> RedirectResponse:
 
 
 if __name__ == "__main__":
+    if not OUTPUT_DIR.exists():
+        print(f"Creating output directory at {OUTPUT_DIR}")
+        OUTPUT_DIR.mkdir(parents=True, exist_ok=True)
+
+    if not os.environ.get("COMMAND"):
+        print(
```

Comment:
I like this! Although thoughts on whether it should be a hard error?

Reply:
I had this as a warning so that you can still use the UI to view existing runs without needing to configure anything.